### PR TITLE
Object_RU1, Object_MA1, and warp pad objects Decompiled

### PR DESCRIPTION
**Pull request by louist103**
Also fixed `TLUTettes` in `Object_Link_Child`

### CODE REVIEW

File: assets/xml/objects/object_ru1.xml [hunk 1]
```diff
@@ -1,69 +1,80 @@
...
+
+        <Texture Name="gRutoChildBlueSkinGradientTex" OutName="blue_skin_gradient" Format="rgb5a1" Width="8" Height="8" Offset="0xE7B8"/>
+
+        <Texture Name="gRutoChildMouthClosedTex" OutName="mouth_closed" Format="ci8" Width="32" Height="32" Offset="0xE838"/>
+        <Texture Name="gRutoChildMouthFrownTex" OutName="mouth_frown" Format="ci8" Width="32" Height="32" Offset="0xFA38"/>
+        <Texture Name="gRutoChildMouthOpenTex" OutName="mouth_open" Format="ci8" Width="32" Height="32" Offset="0x10638"/>
+
+        <Texture Name="gRutoChildEarTex" OutName="ear" Format="rgb5a1" Width="16" Height="16" Offset="0xEC38"/>
+        <Texture Name="gRutoChildBlueSkinPatchesTex" OutName="blue_skin_patches" Format="rgb5a1" Width="16" Height="16" Offset="0xEE38"/>
+        <Texture Name="gRutoChildUnkHeadCircle" OutName="tex_0000F038" Format="rgb5a1" Width="16" Height="16" Offset="0xF038"/> <!-- TODO rename me-->
```
Comment (fig02) [inline assets/xml/objects/object_ru1.xml:55]: Tex

Comment (fig02) [reply assets/xml/objects/object_ru1.xml:55]: looks like you had the same issues in multiple files. i pointed out the one in ru1 and you fixed it in ma1 lol

**Review by fig02 — APPROVED** [review]

File: assets/xml/objects/object_ru1.xml [hunk 2]
```diff
@@ -1,69 +1,83 @@
...
+        <DList Name="gRutoChildLeftThighDL" Offset="0xC188"/>
+        <DList Name="gRutoChildLeftShinDL" Offset="0xC3D8"/>
+        <DList Name="gRutoChildLeftFootDL" Offset="0xC630"/>
+        <DList Name="gRutoChildRightThighDL" Offset="0xC7D0"/>
+        <DList Name="gRutoChildRightShinDL" Offset="0xCA20"/>
+        <DList Name="gRutoChildRightFootDL" Offset="0xCC78"/>
+        <DList Name="gRutoChildChestDL" Offset="0xCE18"/>
+        <DList Name="gRutoChildRightUpperArmDL" Offset="0xD0B8"/>
+        <DList Name="gRutoChildRightFinDL" Offset="0xD308"/>
+        <DList Name="gRutoChildDLRightHand" Offset="0xD5E0"/>
```
[the remark this answers is not in the document]
Comment (EllipticEllipsis) [reply assets/xml/objects/object_ru1.xml:36]: `gRutoChildRightHandDL`, I think?

**Review by fig02 — APPROVED** [review]

File: assets/xml/objects/object_ru1.xml [hunk 3]
```diff
@@ -1,69 +1,83 @@
...
+        <DList Name="gRutoChildLeftThighDL" Offset="0xC188"/>
+        <DList Name="gRutoChildLeftShinDL" Offset="0xC3D8"/>
+        <DList Name="gRutoChildLeftFootDL" Offset="0xC630"/>
+        <DList Name="gRutoChildRightThighDL" Offset="0xC7D0"/>
+        <DList Name="gRutoChildRightShinDL" Offset="0xCA20"/>
+        <DList Name="gRutoChildRightFootDL" Offset="0xCC78"/>
+        <DList Name="gRutoChildChestDL" Offset="0xCE18"/>
+        <DList Name="gRutoChildRightUpperArmDL" Offset="0xD0B8"/>
+        <DList Name="gRutoChildRightFinDL" Offset="0xD308"/>
+        <DList Name="gRutoChildDLRightHandDL" Offset="0xD5E0"/>
```
Comment (Roman971) [inline assets/xml/objects/object_ru1.xml:36]: I think this was missed in the comments from my previous review
```suggestion
        <DList Name="gRutoChildRightHandDL" Offset="0xD5E0"/>
```

**Review by Roman971 — APPROVED** [review]

**Pull request merged**